src/app/states/prerecording.service.ts

```diff
@@ -44,4 +47,26 @@ export class PrerecordingService {
     recordings[index] = { content };
     this.config.save(this.recordingsKey, recordings);
   }
+
+  delete(index: number) {
+    const recordings = this.config.get<Recording[]>(this.recordingsKey) || [];
+    recordings.splice(index, 1);
+    this.config.save(this.recordingsKey, recordings);
+  }
+
+  get(index: number): Recording {
```

Comment:
should not be needed anymore, after applying the other measures.

Reply:
You mean the `delete()` function? Note that with "Developer" toggle enabled, it's still possible to delete prerecordinsg through the UI...

Reply:
the get function